src/main/java/com/redhat/cloud/notifications/routers/NotificationService.java

```diff
@@ -131,47 +128,19 @@ private String getAddress(Principal principal) {
         return apps.getEventTypes(query, applicationIds, bundleId);
     }
 
-    // TODO [BG Phase 2] Delete this method
-    @GET
-    @Path("/eventTypes/affectedByRemovalOfEndpoint/{endpointId}")
-    @Produces(APPLICATION_JSON)
-    @RolesAllowed(RbacIdentityProvider.RBAC_READ_NOTIFICATIONS)
-    public Uni<List<EventType>> getEventTypesAffectedByEndpointId(@Context SecurityContext sec, @PathParam("endpointId") UUID endpointId) {
-        RhIdPrincipal principal = (RhIdPrincipal) sec.getUserPrincipal();
-
-        Multi<EventType> directlyAffected = apps.getEventTypesByEndpointId(principal.getAccount(), endpointId)
-                .onItem().transformToMulti(Multi.createFrom()::iterable);
-        Multi<EventType> indirectlyAffected = resources.getEndpointsPerType(principal.getAccount(), EndpointType.DEFAULT, null, null)
-                .onItem().transformToMulti(Multi.createFrom()::iterable)
-                .toUni().onItem().transformToMulti(defaultEndpoint ->
-            resources.endpointInDefaults(principal.getAccount(), endpointId)
-                    .onItem().transformToMulti(exists -> {
-                        if (exists) {
-                            return apps.getEventTypesByEndpointId(principal.getAccount(), defaultEndpoint.getId())
-                                    .onItem().transformToMulti(Multi.createFrom()::iterable);
-                        }
-
-                        return Multi.createFrom().empty();
-                    })
-        );
-
-        return Multi.createBy().concatenating().streams(directlyAffected, indirectlyAffected)
-                .collect().asList();
-    }
-
     /*
      * Called by the UI to build the endpoint removal confirmation screen.
      * That screen shows all the event types (and their application) that will be affected by the endpoint removal.
      */
     @GET
-    @Path("/bg/eventTypes/affectedByRemovalOfEndpoint/{endpointId}") // TODO [BG Phase 2] Remove '/bg' path prefix
+    @Path("/eventTypes/affectedByRemovalOfEndpoint/{endpointId}")
     @Produces(APPLICATION_JSON)
     @Operation(summary = "Retrieve the event types affected by the removal of an integration.")
     @Tag(name = OApiService.PRIVATE)
```

Comment:
Could you remove these too?

Reply:
Do you mean that `GET /eventTypes/affectedByRemovalOfEndpoint/{endpointId}` is no longer needed after the behavior group migration?